.github/workflows/cd.yml

```diff
@@ -6,7 +6,7 @@ on:
 
 env:
   # mark:automatic-version
-  VERSION: "1.20231001.0"
+  VERSION: "20231001.0.0"
```

Comment:
I think it makes more sense to have our API version as major -- that way we can bump the minor for new backwards-compatible features, and bump the patch for bug fixes and such.

Reply:
This one is controversial.
1. I think it will make it harder for marketing, we call it v1 and v2 everywhere but the actual released version will be different;
2. Tracking breaking changes as a minor is not semver (which is not very important) but not being able to see from a version that there was a breaking change is a downshift in developer experience. Compare this two cases:
```
1.20231001.0
1.20231002.0
1.20231003.0
2.20231004.0
2.20231005.0
```
and
```
20231001.0.0
20231002.0.0
20231003.0.0
20231004.1.0
20231005.0.0
```
^ in 2nd it's way harder to tell if there were breaking changes between `20231001.0.0` and `20231005.0.0`. Or do you want to use a minor as a major and never reset it to zero?

Reply:
Yeah, we would need to solve the marketing aspect.

I was thinking the versions would be more like

```
20231001.0.0
20231001.1.0
20231001.2.0
20231001.3.0
20231001.4.0

20240508.0.0
...

20241112.0.0
...
```

Reply:
We don't (won't?) have a `v2`

Reply:
Yeah, so we will lose the knowledge that there was a breaking change every day, which will be not the best for developer experience in a slightly larger team than we have now.

Reply:
@AndrewDryga I think there may be some misconceptions with how the versioning works with #2041 

- `MAJOR` is the API version date, e.g. `20231001`. The only changes when we release a breaking change. This is essentially our REST API and Connlib version, so all connected entities (clients, gateways, relays) on `20231001` can be expected to work together.
- `MINOR` includes feature updates and bumps sequentially for each patch
- `PATCH` includes 

We're stuck using semver because every package manager (and app store distribution) expects it.

My proposal was to use the API version as MAJOR since the `1` I had before has no significance in the system. When will we release v2? etc

Reply:
With the work from #2203, clients will user that they need to update their version, and gateways/relays can send email notifications to the admin alerting them the same.

Reply:
Okay, I'm sold 👍 😄.

Reply:
Hm, yeah, that's how it was before -- I can revert it back.

Reply:
👍 for doing that. We won't need that "feature" part of the version anyways with date-versioning, and patches will be just for builds within same date.